### DISCUSSION TITLE
Net worth missing on V2 home screen

### DISCUSSION THREAD

**Discussion by matejdro** [firefly-iii/firefly-iii#4772] [General]
Is that intentional?

Debug information generated at 2021-05-15 05:05:36 UTC for Firefly III version **5.5.10**.

| Scope | Version |
| --- | --- |
| Firefly III | 5.5.10 |
| Firefly III API | 1.5.2 |
| PHP | 8.0.3 |
| Host | Linux |

| System info | Value |
| --- | --- |
| Installation ID | 1c789eff-ec96-4172-80d7-9ab04aabfda8 |
| Using docker? | true |
| Telemetry | false |
| Layout | v2 |
| System TZ |  |
| Browser TZ | Europe/Ljubljana |
| App environment | local |
| App debug mode | false |
| App cache driver | file |
| App logging | info, stack |
| Display errors | Off |
| Error reporting | ALL errors |
| Interface | apache2handler |
| Default language | en_US |
| Default locale | equal |
| BCscale | 24 |
| DB drivers | mysql, pgsql, sqlite |
| Current driver | pgsql |
| DB version | 16 (exp. 16) |
| Login provider | eloquent  |
| Trusted proxies (.env) | ** |

| User info | Value |
| --- | --- |
| Session start | 2021-05-01 00:00:00 |
| Session end | 2021-05-31 23:59:59 |
| Session first | 2017-12-17 00:00:00 |
| User ID | 1 |
| User language | en_US |
| User locale | en_US |
| Attempt at "en_US.utf8" | 'en_US.utf8' |
| Attempt at "en_US.UTF-8" | 'en_US.UTF-8' |
| User agent | Mozilla/5.0 (X11; Linux x86_64; rv:87.0) Gecko/20100101 Firefox/87.0 |

**Comment by JC5** [comment]
Only if there's nothing to show.

**Reply by matejdro** [reply]
It seems like upgrading to ` 5.5.11` fixed it. It is shown again. Before, entire top row was gone (Balance, Left to spend and Net worth boxes were gone, Your accounts chart was on the top).

**Reply by matejdro** [reply]
I see it was already reported: https://github.com/firefly-iii/firefly-iii/issues/4729
